indico/modules/events/templates/display/_common.html

```diff
@@ -0,0 +1,104 @@
+{% macro _render_location_tip(item) -%}
+    {% set show_url = item.room and item.room.map_url %}
+    <div class="location-info-tip">
+        {% if item.room_name -%}
+            <h3>{{ item.room_name }}</h3>
+        {% endif %}
+        {% if item.venue_name -%}
+            <h4>{{ item.venue_name }}</h4>
+        {% endif %}
+        {% if item.address %}
+            <div class="location-address">
+                {#- -#}
```

Comment:
You can use `{{- item.address }}` to get rid of the whitespace - no need for the comment "hack" (only needed if there are no other jinja tags around)